CRM/MembershipExtras/Queue/Builder/MultipleInstalmentPlans.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use CRM_MembershipExtras_Queue_Builder_PaymentPlans as PaymentPlansQueueBuilder;
+
+class CRM_MembershipExtras_Queue_Builder_MultipleInstalmentPlans extends PaymentPlansQueueBuilder {
```

Comment:
Should we have the word "OfflineAuto" in the class or name space as same as period class? So we know that they are